src/providers/ethers.tsx

```diff
@@ -1,19 +1,69 @@
-import React, { createContext, useMemo, useContext } from 'react';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React, { createContext, useEffect, useContext, useState } from 'react';
 import { SafeAppsSdkProvider } from '@gnosis.pm/safe-apps-ethers-provider';
 import { useSafeAppsSDK } from '@gnosis.pm/safe-apps-react-sdk';
+import { fetch } from 'cross-fetch';
+import { setPtClient, setLbClient } from '../ptGraphClient';
+import { HttpLink, InMemoryCache, ApolloClient } from '@apollo/client';
 
 const EthersContext = createContext<SafeAppsSdkProvider | undefined>(undefined);
 
+const PT_GRAPH_ENDPOINTS: any = {
```

Comment:
if you open the app in a safe connected to the mainnet  it should now pull in from the mainnet graph, and if you open the app in rinkeby it'll connect to the rinkeby graph.